.github/workflows/feature-requests.yml

```diff
@@ -0,0 +1,15 @@
+name: Feature request triage bot
+
+on:
+  schedule:
+    # Run at 13:00 every day
+    - cron: '0 13 * * *'
+
+jobs:
+  feature_triage:
+    if: github.repository == 'angular/angular-cli'
```

Comment:
Is the motivation for this to disable the workflow for forks? If so, we should probably put a comment calling that out as it isn't very obvious. Someone would reasonably thinking that this is always `true` and not consider the forking use case.

Reply:
Added.